pkg/sfu/downtrack.go

```diff
@@ -130,6 +130,7 @@ type DownTrack struct {
 	maxPacketTs uint32
 
 	bandwidthConstrainedMuted atomicBool
+	maxAvailableLayer         atomicInt32
```

Comment:
Guess this needs to be ported to the latest code.

I am thinking with the latest stream allocator effort, there is going to be enough perturbance.

Reply:
I checked the latest, and some form of this is already available in Forwarder. @shishirng is working on computing MOS for the next version of connection quality. and we'll work to integrate the new changes into it